llsd/base.py

```diff
@@ -317,7 +317,7 @@ def _array_to_python(node, depth=0):
 def _to_python(node, depth=0):
     "Convert node to a python object."
     if depth > MAX_PARSE_DEPTH:
-        raise LLSDParseError("Cannot serialize depth of more than %d" % MAX_FORMAT_DEPTH)
+        raise LLSDSerializationError("Cannot serialize depth of more than %d" % MAX_FORMAT_DEPTH)
```

Comment:
Isn't `_to_python()` used by parsers? The docstring seems to suggest that we're deserializing rather than serializing. Even if I'm wrong about that, though, lines 319 and 320 are still inconsistent.